src/Pim/Bundle/VersioningBundle/Doctrine/ORM/VersionRepository.php

```diff
@@ -28,7 +28,7 @@ public function getLogEntries($resourceName, $resourceId)
     {
         return $this->findBy(
             ['resourceId' => $resourceId, 'resourceName' => $resourceName, 'pending' => false],
-            ['loggedAt' => 'desc']
+            ['version' => 'desc']
```

Comment:
It does the same thing basically it just enhances the precision because loggedAt versions can have the same datetime (but not the same version number).